lib/ruby_indexer/lib/ruby_indexer/declaration_listener.rb

```diff
@@ -45,6 +45,11 @@ def initialize(index, dispatcher, parse_result, file_path)
         :on_constant_or_write_node_enter,
         :on_constant_and_write_node_enter,
         :on_constant_operator_write_node_enter,
+        :on_instance_variable_write_node_enter,
+        :on_instance_variable_and_write_node_enter,
+        :on_instance_variable_operator_write_node_enter,
+        :on_instance_variable_or_write_node_enter,
```

Comment:
My reasoning was that it would be useful for developers to know all the places where the instance variable is set to something.

Especially since a lot of Ruby code doesn't set ivars in initialize, but rather somewhere else. So, unlike with Sorbet that enforces you declare them, we don't necessarily have a single place where the ivar is declared. So it made sense to me to just show everywhere it is set to something.